arch/risc-v/src/common/riscv_internal.h

```diff
@@ -116,6 +116,10 @@ EXTERN uint32_t g_intstackalloc; /* Allocated stack base */
 EXTERN uint32_t g_intstacktop;   /* Initial top of interrupt stack */
 #endif
 
+#ifdef CONFIG_SMP
+EXTERN volatile bool g_serial_ok;
```

Comment:
only k210 really use g_serial_ok, so it's better to move it to k210 specific header file instead.

Reply:
Yes, only k210 need it now, but other smp system also need it, qemu-rv smp support is ready to submit.